task_5_1.py

```diff
@@ -0,0 +1,19 @@
+import math
+
+precision = 4
+
+
+def series_sum(x, precision):
```

Comment:
you should name your functions as an actions.
**series_sum** is set of nouns. on the other hand **get_series_sum** or **calculate_series_sum** are actions